src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -9,24 +9,17 @@ public enum DocumentFlags
 
         Artificial = 0x1,
         Versioned = 0x2,
-        Reserved1 = 0x4,
-        Reserved2 = 0x8,
+        Revision = 0x4,
+        Reserved = 0x8,
 
         FromIndex = 0x10,
-        FromVersionStorage = 0x20,
+        Reserved1 = 0x20,
         FromReplication = 0x40,
-        Reserved3 = 0x80,
+        FromSmuggler = 0x80,
 
-        HasAttachments = 0x100
-    }
-
-    [Flags]
-    public enum NonPersistentDocumentFlags
```

Comment:
@fitzchak can you consult this change with @grisha-kotler ? I think that the whole idea was to not persist those flags.